README.md

```diff
@@ -122,6 +122,28 @@ Templates have these goals:
 Copier tries to have a smooth learning curve that lets you create simple templates that
 can evolve into complex ones as needed.
 
+## Templates versions
```

Comment:
These docs look great, although I don't think this subject is so important as to document it directly in the README. It's more an implementation/usage detail that is only useful when you start using copier more deeply.

There are other places where this would fit better:

- https://copier.readthedocs.io/en/latest/generating/
- https://copier.readthedocs.io/en/latest/updating/
- https://copier.readthedocs.io/en/latest/configuring/#vcs_ref (BTW, here it mentions something related):

  > By default, copier will copy from the last release found in template git tags, sorted as PEP 440.

Please pick the one you think fits best and move these docs there.

Reply:
This versioning behavior tripped up both myself and a couple of my colleages on unrelated occasions before I figured out what was going on. 
It seemed particularly confusing when following an example of using a local clone of a template - the assumed behavior is that the template that's checked out in the specified folder would be used as-is, rather than copier using something from its git history.
As such I feel it does warrant being in a fairly prominent place. 

That being said, perhaps the "Generating a project" docs page just after the first initial usage section makes sense. 
Also, with the print() noting the version used when running, that should inform users quickly enough to avoid future confusion.

To be honest I don't think I'd read any of those docs previously... the readme felt like it had enough information really. When I started to get confused by it I really should have read through the docs, maybe I might have found that note on the `--vcs-ref` arg.

Reply:
The 1st part you say was #184 and got fixed recently. I'm not sure if it was released already though.



> That being said, perhaps the "Generating a project" docs page just after the first initial usage section makes sense.
> Also, with the print() noting the version used when running, that should inform users quickly enough to avoid future confusion.

I agree. Would you please move this there then please?

Reply:
Oh sorry, I didn't notice you already did it.